src/content/about/metrics.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Metrics
+weight: 4
+---
+
+- [Per Worker Script Metrics](#per-worker-script-metrics)
+  - [Invocation statuses](#invocation-statuses)
+  - [Worker requests](#worker-requests)
+  - [CPU time](#cpu-time)
+- [Per Zone Metrics](#per-zone-metrics)
+  - [Total requests](#total-requests)
+  - [Status codes](#status-codes)
+  - [Bandwidth](#bandwidth)
+  - [Subrequests](#subrequests)
+- [GraphQL](#graphql)
+
+Workers metrics show performance and usage of your Workers and help diagnose issues.
+
+Workers Metrics are available in Per Zone and Per Worker script contexts.
+
+## Per Worker Script Metrics
+
+Aggregates request data for an individual Worker script across all zones including your workers.dev subdomain. On your [Workers dashboard](https://dash.cloudflare.com/?account=workers/overview), click on any Worker to view its metrics.
+
+Per Worker script metrics can be inspected by time ranges within the last 30 days. The dashboard includes the charts and information described below.
+
+### Invocation statuses
+
+Worker invocation statuses are generated by the Workers runtime and indicate whether a Worker script executed successfully or failed to generate a response. Invocation statuses differ from HTTP status codes. In some cases, a Worker script invocation succeeds but does not generate a successful HTTP status because of another error encountered outside of the Workers runtime. Some invocation statuses result in a [Workers error code](/about/tips/debugging/#error-pages-generated-by-workers) being returned to the client.
```

Comment:
Can we specific what error codes/ HTTP status codes will be return by each error via two columns in the table? That way users won't have to guess what of the same invocation statuses resulted with what. and it saves 3 sentences

Reply:
Worker error codes that match 1:1 with invocation statuses are in the `Worker error codes` column.  Unfortunately, there's no way to reliably map worker invocation codes with HTTP codes because an error may succeed but the request fails somewhere else, or conversely, the worker may fail, but something else in the system returns a 200 to the eyeball.